awx/main/models/jobs.py

```diff
@@ -1140,22 +1140,58 @@ class SystemJobOptions(BaseModel):
     Common fields for SystemJobTemplate and SystemJob.
     '''
 
-    SYSTEM_JOB_TYPE = [
-        ('cleanup_jobs', _('Remove jobs older than a certain number of days')),
-        ('cleanup_activitystream', _('Remove activity stream entries older than a certain number of days')),
-        ('cleanup_sessions', _('Removes expired browser sessions from the database')),
-        ('cleanup_tokens', _('Removes expired OAuth 2 access tokens and refresh tokens'))
-    ]
-
     class Meta:
         abstract = True
 
+    SYSTEM_CLEANUP_JOBS = (
+        'cleanup_jobs',
+        _('Remove jobs older than a certain number of days')
+    )
+    SYSTEM_CLEANUP_ACTIVITY = (
+        'cleanup_activitystream',
+        _('Remove activity stream entries older than a certain number of days')
+    )
+    SYSTEM_CLEANUP_SESSIONS = (
+        'cleanup_sessions',
+        _('Removes expired browser sessions from the database')
+    )
+    SYSTEM_CLEANUP_TOKENS = (
+        'cleanup_tokens',
+        _('Removes expired OAuth 2 access tokens and refresh tokens')
+    )
+
+    SYSTEM_JOB_TYPES = (
+        SYSTEM_CLEANUP_JOBS,
+        SYSTEM_CLEANUP_ACTIVITY,
+        SYSTEM_CLEANUP_SESSIONS,
+        SYSTEM_CLEANUP_TOKENS,
+    )
+    CONFIGURABLE_RETENTION_TYPES = (
+        SYSTEM_CLEANUP_JOBS,
+        SYSTEM_CLEANUP_ACTIVITY,
```

Comment:
The `default_days` field is mutable for system job template types placed in this tuple.